### DISCUSSION TITLE
Ignore keyboard layout

### DISCUSSION THREAD

**Discussion by SET001** [Leafwing-Studios/leafwing-input-manager#144] [Q&A]
Right now controls work only when English layout is selected. If you would switch to any other than English keyboard layout, then bindings like `KeyCode::A `, `KeyCode::S`,`KeyCode::D` will not work. Is there any workaround for that?

**Comment by alice-i-cecile** [comment]
This requires access to ScanCode information. This is not currently very feasible and requires work upstream in both Bevy and winit.

**Comment by SET001** [comment]
That is pretty frustrating but thank for your response and for your work on this lib

**Reply by alice-i-cecile** [reply]
You're very welcome. It's something I want too; but this isn't the right place to solve it.

**Comment by it-me-joda** [comment]
This is no longer an issue. I'm unsure of exactly when this was resolved but I'm on dvorak and the mapping just works as if I'm on qwerty.
